Sources/MuxUploadSDK/PublicAPI/MuxUpload.swift

```diff
@@ -153,12 +328,14 @@ public final class MuxUpload : Hashable, Equatable {
     /**
     URL to the file that will be uploaded
      */
-    public var videoFile: URL { get { return uploadInfo.videoFile } }
+    public var videoFile: URL { get { return uploadInfo!.videoFile } }
```

Comment:
Thanks for noting this, the property is now optional so there is no longer a need for a force-unwrap / `!` usage.

This change is required as a result of the new MuxUpload `init` methods introduced in this PR that allow it to be initialized with an arbitrary `AVAsset`. `MuxUpload` is now no longer guaranteed to have a local input URL across its entire lifecycle, because `AVAsset` isn't guaranteed to have a local `file://` URL. In certain cases it may have a remote URL and at other times it may not have an available URL at all.

AVAsset itself doesn't have a `url` property. (It's subclass `AVURLAsset` does expose a `url` but not every `AVAsset` is an `AVURLAsset`, so casting to a subclass may not always succeed.)

When `MuxUpload` is ready to be sent over the network it is still required to have a local URL, so that aspect of the SDK implementation shouldn't need any changes. But when in `ready`, `started`, `preparing` states it won't have it.

I'll also update the API docs to call this out.